cypress/e2e/user.cy.js

```diff
@@ -2,11 +2,23 @@
 /// <reference types="../support" />
 
 describe('User', () => {
-  before(() => {
+   let firstUser;
+   let secondUser;
 
-  });
+   before(() => {
+      cy.task('db:clear');
+      cy.register().then(user => {
+         firstUser = user;
+      });
+      cy.register().then(user => {
+         secondUser = user;
+      });
+   });
 
-  it.skip('should be able to follow the another user', () => {
-
-  });
+   it('should be able to follow the another user', () => {
+      cy.login(firstUser);
+      cy.wait(2000);
```

Comment:
Avoid using hardcoded wait times. Instead, use Cypress commands like 'cy.intercept()' to wait for specific API calls to complete.